packages/venia-concept/package.json

```diff
@@ -47,14 +44,14 @@
     "apollo-link-http": "~1.5.11",
     "braintree-web-drop-in": "~1.16.0",
     "graphql-cli": "~3.0.11",
-    "graphql-cli-validate-magento-pwa-queries": "~1.0.0",
+    "graphql-cli-validate-magento-pwa-queries": "~1.1.0-beta.1",
```

Comment:
Was this change for testing purposes or do we actually want to commit it?

Reply:
For testing. Removing it.